src/main/java/lotto/domain/LottoNoPool.java

```diff
@@ -4,25 +4,18 @@
 import java.util.Map;
 
 public class LottoNoPool {
-    private final Map<Integer, LottoNo> lottoNoPool;
+    private static final Map<Integer, LottoNo> pool = new HashMap<>();
```

Comment:
미리 캐싱해서 갖고있어도 될 것 같아요 :)
```suggestion
    private static final Map<Integer, LottoNo> pool = 
                                       IntStream.rangeClosed(LottoNo.MIN, LottoNo.MAX)
                                                  .boxed()
                                                  .collect(Collectors.toMap(Function.identity(), LottoNo::new));
```